internal/step/plugin/provider_test.go

```diff
@@ -32,13 +33,13 @@ func (s *deployFailStageChangeHandler) OnStepComplete(
 		panic(fmt.Errorf("invalid previous stage: %s", previousStage))
 	}
 	if previousStageOutputID == nil {
-		panic(fmt.Errorf("no previous stage output ID"))
+		panic(fmt.Errorf("no previous stage output Name"))
```

Comment:
Name? ID? Why did "stage output" terminology change?

Also 39, 42, 68, 71... and others